### ISSUE TITLE
Migrating myst node problem

### ISSUE THREAD

**Issue by samujele** [mysteriumnetwork/node#5164]
Hi, I have problem with running my node on a linux VM on hyperV server.
It ran perfectly right until linux decided to crash.. I resurected the VM, saved the /var/lib/mysterium-node folder, and then copied that over existing folder, and then even reinstalled linux side by side to the old one, instaled myst-node then stopped service and copied over that folder. But when I try to run the service it returns Main PID: 1337 (code=exited, status=2) but no further explanation. Os: Ubuntu 20.04.3

**Comment by tomasmik** [comment]
Did you completely replace that directory with the one you copied or copied files on top of what was created by a fresh node?

**Comment by samujele** [comment]
I tried both metods,did not work. Stopped service of working new node then
did that... As sudo, ofc... Even did the chown -R mysterium-node on pasted
directory...

**Comment by etherunit** [comment]
What's your node identity @samujele ?

**Comment by samujele** [comment]
0x957cb6324b25cedfb77ece353aac05b0065be69f

**Comment by etherunit** [comment]
thanks. couple of things to clarify:
check service status:
`sudo systemctl status mysterium-node.service`

extract node logs (if possible) at the time failure happens.
https://docs.mysterium.network/troubleshooting/reporting-an-issue

**Comment by samujele** [comment]
service status pasted on pastebin: https://pastebin.com/N9Se4Urr

But the curl command returned : curl: (7) Failed to connect to localhost port 4050: Connection refused
I assume it cant get anything because service can't get running.

**Comment by etherunit** [comment]
That is correct. Service has failed to activate and it's currently not running. I suspect that migration process has been incomplete and part of the data files are missing.
